apps/studio/src/pages/home/library/MarketplaceTab/CreateSale.tsx

```diff
@@ -98,13 +99,25 @@ export const CreateSale = () => {
       ),
     totalSaleValue: Yup.number()
       .required("This field is required")
+      .typeError("")
```

Comment:
Was added as there was a non UX friendly validation error outputting when `-` or `.` was entered on its own into the input field. The field is already of type number so other symbols/characters are restricted anyway.

Validation error:
`totalSaleValue must be a `number` type, but the final value was: `NaN` (cast from the value `NaN`).`